packages/ncids-css/scss/components/nci-header/internal/_search.scss

```diff
@@ -1,96 +1,89 @@
 // SEARCH STYLES
 .nci-header--extended {
-  .nci-header-nav__secondary {
-    &.search-focused {
-      .nci-header-mobilenav__open-btn {
-        display: none;
-        @include at-media('mobile-lg') {
-          display: block;
-        }
-        @include at-media('desktop') {
-          display: none;
-        }
-      }
-    }
+	.nci-header-nav__secondary {
+		&.search-focused {
+			.nci-header-mobilenav__open-btn {
+				@include u-display('none');
+				@include at-media('mobile-lg') {
+					@include u-display('block');
+				}
+				@include at-media('desktop') {
+					@include u-display('none');
+				}
+			}
+		}
 
-    .nci-header-search {
+    // form wrapping sws input
+		.nci-header-search {
 			flex-grow: 1;
-      margin: 0;
+			margin: 0;
 			white-space: nowrap;
 			position: relative;
+			@include u-maxw(100%);
+      @include u-display('flex');
+      justify-content: center;
 
-      @include at-media('mobile-lg') {
-        flex-grow: 0;
-        min-width: 350px;
-      }
+			@include at-media('mobile-lg') {
+				flex-grow: 0;
+				@include u-minw(300px);
```

Comment:
This is going to be an issue. USWDS does not have a token for this. they would probably want `@include u-minw('mobile')` which is 320px.

Reply:
`SassError: "'mobile' is not a valid USWDS min-width token.`  Mebbe its a v3 token in the documentation?

Reply:
min-width only goes to 15 (like Spinal Tap)